Lib/test/test_mmap.py

```diff
@@ -37,6 +37,7 @@ def test_basic(self):
             f.write(b'\0'* (PAGESIZE-3) )
             f.flush()
             m = mmap.mmap(f.fileno(), 2 * PAGESIZE)
+            self.assertEqual(f.tell(), 2 * PAGESIZE)
```

Comment:
What `f.tell()` returned before this change? What `m.tell()` returned and what it returns now?